first_problem_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test/unit'
+require_relative'./first_problem'
+
+class FirstProblemTest < Test::Unit::TestCase
+
+  # Called before every test method runs. Can be used
+  # to set up fixture information.
+  def setup
+    # Do nothing
```

Comment:
There's no need to have methods that do nothing

Reply:
On the IDE i used it generates test classes with these methods .  
As i don't have plenty ruby experience i assumed that these methods are ok to remain there. However i would normally not have empty methods . 
I will remove them

Reply:
i took liberty to remove these methods from the other branches also